releases/nginx-ingress.yaml

```diff
@@ -73,6 +73,20 @@ releases:
   installed: {{ env "NGINX_INGRESS_INSTALLED" | default "true" }}
   values:
     - controller:
+        livenessProbe:
+          failureThreshold: {{ env "NGINX_INGRESS_LIVENESS_PROBE_FAILURE_THRESHOLD" | default 3 }}
+          initialDelaySeconds: {{ env "NGINX_INGRESS_LIVENESS_PROBE_INITIAL_DELAY_SECONDS" | default 10 }}
+          periodSeconds: {{ env "NGINX_INGRESS_LIVENESS_PROBE_PERIOD_SECONDS" | default 10 }}
+          successThreshold: {{ env "NGINX_INGRESS_LIVENESS_PROBE_SUCCESS_THRESHOLD" | default 1 }}
+          timeoutSeconds: {{ env "NGINX_INGRESS_LIVENESS_PROBE_TIMEOUT_SECONDS" | default 1 }}
```

Comment:
```suggestion
          timeoutSeconds: {{ env "NGINX_INGRESS_LIVENESS_PROBE_TIMEOUT_SECONDS" | default 5 }}
```

Reply:
No problem - it's your repo :)

You should be aware that I took the value 1 from the upstream chart - indeed all the defaults are from there - I did it like this so my change would not affect releases where these new envvars are not set.

https://github.com/helm/charts/blob/master/stable/nginx-ingress/values.yaml#L210-L223